apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -296,44 +299,89 @@ const Template = ({
     `;
 };
 
-const FormTemplate: TemplateFunction<InputProps> = (props: InputProps) => {
-    function onSubmit (event: SubmitEvent) {
-        event.preventDefault();
+const WithLabelTemplate: TemplateFunction<InputProps> = (props: InputProps) => html`
+        <p>Please note, the label is a separate component. See <pie-link href="/?path=/story/form-label">pie-form-field</pie-link>.</p>
+        <pie-form-label for="${props.name}">Label</pie-form-label>
+        ${Template(props)}
+    `;
 
-        const form = event.target as HTMLFormElement;
-        const formData = new FormData(form);
+const ExampleFormTemplate: TemplateFunction<InputProps> = () => html`
+    <style>
+        .form {
+            display: flex;
+            flex-direction: column;
+            padding: var(--dt-spacing-d);
+        }
 
-        const data: { [key: string]: string } = {};
-        formData.forEach((value, key) => {
-            data[key] = value.toString();
-        });
+        .form-field {
```

Comment:
Could be a question for design (not to change anything for this PR). Should we have guidelines for spacing between form elements when used together (and if so, should we try and provide either utility classes or a way of handling this for our components, maybe in `pie-css`)

Reply:
We absolutely should. That's why I'm pushing for form patterns to be documented 👍